nuxeo-services/nuxeo-platform-directory/nuxeo-platform-directory-core/src/main/java/org/nuxeo/ecm/directory/DirectoryServiceImpl.java

```diff
@@ -82,7 +82,12 @@ public void start(ComponentContext context) {
         // their tables are created (outside a transaction) if needed
         for (Directory dir : getDirectories()) {
             // open directory to init its resources (tables for SQLDirectory)
-            dir.getSession().close();
+            try {
+                dir.getSession().close();
+            } catch (Exception e) {
+                log.error(String.format("Could not open session for directory %s. Error message: %s", dir.getName(),
```

Comment:
Could you remove Error message from log message and give to logger the exception ?